Master-OB-OpenAPI.json

```diff
@@ -14799,7 +14802,7 @@
           },
           {
             "type": "object",
-            "description": "Amount paid upfront for signing up in the program",
+            "description": "Amount paid by service requestor for signing up in the program.",
```

Comment:
@kevinkirkl3y - I believe the spelling we used is Requester instead of Requestor.